sound/soc/sof/ipc4-topology.c

```diff
@@ -111,6 +111,12 @@ static const struct sof_topology_token gain_tokens[] = {
 		get_token_u32, offsetof(struct sof_ipc4_gain_data, init_val)},
 };
 
+/* SRC */
+static const struct sof_topology_token src_tokens[] = {
```

Comment:
typo in commit message "This PR adds the prepare and setup ops for SRC"

Reply:
sure, thanks!